third_party/2and3/yaml/__init__.pyi

```diff
@@ -22,10 +22,27 @@ def safe_load_all(stream: Union[str, IO[str]]) -> Iterator[Any]: ...
 def emit(events, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=...): ...
 def serialize_all(nodes, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...): ...
 def serialize(node, stream=..., Dumper=..., **kwds): ...
-def dump_all(documents: Sequence[Any], stream: IO[str]=..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> Any: ...
-def dump(data: Any, stream: IO[str]=..., Dumper=..., **kwds) -> Any: ...
-def safe_dump_all(documents: Sequence[Any], stream: IO[str]=..., **kwds) -> Any: ...
-def safe_dump(data: Any, stream: IO[str]=..., **kwds) -> Any: ...
+
+@overload
+def dump_all(documents: Sequence[Any], stream: None, Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> AnyStr: ...
+@overload
+def dump_all(documents: Sequence[Any], stream: IO[str]=..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> None: ...
+
+@overload
+def dump(data: Any, stream: None, Dumper=..., **kwds) -> AnyStr: ...
+@overload
+def dump(data: Any, stream: IO[str]=..., Dumper=..., **kwds) -> None: ...
+
+@overload
+def safe_dump_all(documents: Sequence[Any], stream: None, **kwds) -> AnyStr: ...
+@overload
+def safe_dump_all(documents: Sequence[Any], stream: IO[str]=..., **kwds) -> None: ...
```

Comment:
The `=...` should go to the other overload; it returns a string, not None, if you omit the stream argument.

Reply:
Fixed by a78cca2a